### PR TITLE
Added cluster service version template

### PR DESCRIPTION
**Pull request by arjunrn**
Signed-off-by: Arjun Naik <anaik@redhat.com>

### CODE REVIEW

File: config/templates/osd-metrics-exporter-csv-template.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: osd-metrics-exporter
```
Comment (bdematte) [inline config/templates/osd-metrics-exporter-csv-template.yaml:4]: I had a look for reference to other operators, and convention is to have {operator-name}-{version} for the name field (like [in aws-account-operator](https://github.com/openshift/aws-account-operator/blob/1671948c909525c5df58e81464755a70b452d426/config/templates/aws-account-operator-csv-template.yaml#L4) )

File: config/templates/osd-metrics-exporter-csv-template.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: osd-metrics-exporter-0.0.1
```
Comment (bdematte) [inline config/templates/osd-metrics-exporter-csv-template.yaml:4]: it is not allowed with the verions lower which is 0.0.2 :)

Comment (arjunrn) [reply config/templates/osd-metrics-exporter-csv-template.yaml:4]: This is just a template which is used to generate the actual CSV and the name gets set [like so](https://github.com/openshift/osd-metrics-exporter/blob/6f33fc791d60572742b3bc0bfc67076cbe9e7fa4/hack/generate-operator-bundle.py#L95).

**Pull request merged**